### ISSUE TITLE
is_encrypted on unprotected PPT gives error

### ISSUE THREAD

**Issue by ha1f3mpty** [nolze/msoffcrypto-tool#47]
When evaluating a password protected PPT there is no error and it looks as though everything happens as expected 
This (run against a known password protected PPT):
```
filePPT = r'F:\Decepticons-office-pw.ppt'
print(msoffcrypto.OfficeFile(open(filePPT, "rb")).is_encrypted())
if (msoffcrypto.OfficeFile(open(filePPT, "rb")).is_encrypted()) == 0:
    print("File is not password protected.")
else:
    print("File is password protected.")
```

Produces:
```
C:\Python\Python38-32\python.exe F:/TestOfficePassword.py
True
File is password protected.

Process finished with exit code 0
```

This (run againsta a known unprotected PPT):
```
filePPT = r'F:\Decepticons-office.ppt'
print(msoffcrypto.OfficeFile(open(filePPT, "rb")).is_encrypted())
if (msoffcrypto.OfficeFile(open(filePPT, "rb")).is_encrypted()) == 0:
    print("File is not password protected.")
else:
    print("File is password protected.")
```

Produces:
```
C:\Python\Python38-32\python.exe F:/TestOfficePassword.py
Traceback (most recent call last):
  File "F:/TestOfficePassword.py", line 12, in <module>
    print(msoffcrypto.OfficeFile(open(filePPT, "rb")).is_encrypted())
  File "C:\Python\Python38-32\lib\site-packages\msoffcrypto\format\ppt97.py", line 756, in is_encrypted
    usereditatom = _parseUserEditAtom(self.data.powerpointdocument)
  File "C:\Python\Python38-32\lib\site-packages\msoffcrypto\format\ppt97.py", line 234, in _parseUserEditAtom
    encryptSessionPersistIdRef, = unpack("<I", blob.read(4))
struct.error: unpack requires a buffer of 4 bytes

Process finished with exit code 1
```

PPTX appear to function correctly for both protected and unprotected files.

Thank you in advance for any help.

**Comment by nolze** [comment]
Thank you for reporting!

I found that this is a bug in the library and should be fixed in #49.
I'll ship it in the next release; until then, please use the upstream by `pip install -U git+https://github.com/nolze/msoffcrypto-tool`.
It would be helpful if you let me know if the fix works fine.

**Comment by ha1f3mpty** [comment]
IT WORKS!!

This:
```
filePPT = r'F:\Decepticons-office.ppt'
print(msoffcrypto.OfficeFile(open(filePPT, "rb")).is_encrypted())
if (msoffcrypto.OfficeFile(open(filePPT, "rb")).is_encrypted()) == 0:
    print("File is not password protected.")
else:
    print("File is password protected.")
```

Produces:
```
C:\Python\Python38-32\python.exe F:/TestOfficePassword.py
False
File is not password protected.

Process finished with exit code 0
```
Thank you very much for the quick response.
